src/overlays/actors/ovl_En_Kusa/z_en_kusa.c

```diff
@@ -7,72 +16,485 @@
+void func_80A9C164(Actor* thisx, GlobalContext* globalCtx);
 
-/*
-const ActorInit En_Kusa_InitVars = {
-    ACTOR_EN_KUSA,
-    ACTORTYPE_PROP,
-    FLAGS,
-    OBJECT_GAMEPLAY_KEEP,
-    sizeof(EnKusa),
-    (ActorFunc)EnKusa_Init,
-    (ActorFunc)EnKusa_Destroy,
-    (ActorFunc)EnKusa_Update,
-    NULL,
+void func_80A9AFA0(EnKusa* this, EnKusaActionFunc actionFunc);
+s32 func_80A9AFAC(EnKusa* this, GlobalContext* globalCtx, f32 arg2);
+void func_80A9B07C(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9B140(EnKusa* this);
+void func_80A9B174(Vec3f* this, f32 arg1);
+void func_80A9B1FC(EnKusa* this);
+void func_80A9B21C(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9B574(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9B630(Actor* thisx, GlobalContext* globalCtx);
+
+void func_80A9B7EC(EnKusa* this);
+void func_80A9B810(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9B89C(EnKusa* this);
+void func_80A9B8D8(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9BA98(EnKusa* this);
+void func_80A9BAD8(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9BBB0(EnKusa* this);
+void func_80A9BC1C(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9BEAC(EnKusa* this);
+void func_80A9BEFC(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9BF30(EnKusa* this, EnKusaActionFunc actionFunc);
+void func_80A9BF3C(EnKusa* this);
+void func_80A9BFA8(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9C00C(EnKusa* this);
+void func_80A9C068(EnKusa* this, GlobalContext* globalCtx);
+void func_80A9C164(Actor* thisx, GlobalContext* globalCtx);
+
+s16 D_80A9C1D0[] = { 0x00000000 };
+
+s16 D_80A9C1D4[] = { 0x00000000 };
+
+s16 D_80A9C1D8[] = { 0x00000000 };
+
+s16 D_80A9C1DC[] = { 0x00000000 };
+
+const ActorInit En_Kusa_InitVars = { ACTOR_EN_KUSA,
+                                     ACTORTYPE_PROP,
+                                     FLAGS,
+                                     OBJECT_GAMEPLAY_KEEP,
+                                     sizeof(EnKusa),
+                                     (ActorFunc)EnKusa_Init,
+                                     (ActorFunc)EnKusa_Destroy,
+                                     (ActorFunc)EnKusa_Update,
+                                     NULL };
+
+s16 D_80A9C200[] = { OBJECT_GAMEPLAY_FIELD_KEEP, OBJECT_KUSA, OBJECT_KUSA };
+
+// sCylinderInit
+static ColliderCylinderInit D_80A9C208 = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x29, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x4FC00758, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
+    { 12, 44, 0, { 0, 0, 0 } },
+};
+
+// Collision Check data
+// sColChkInfoInit
+CollisionCheckInfoInit D_80A9C234 = { 0, 0xC, 0x1E, 0xFF };
+
+Vec3f D_80A9C23C[] = { { 0.0, 0.707099974155, 0.707099974155 },
+                       { 0.707099974155, 0.707099974155, 0.0 },
+                       { 0.0, 0.707099974155, -0.707099974155 },
+                       { -0.707099974155, 0.707099974155, 0.0 } };
```

Comment:
```suggestion
static Vec3f D_80A9C23C[] = { { 0.0f, 0.7071f, 0.7071f },
                       { 0.7071f, 0.7071f, 0.0f },
                       { 0.0f, 0.7071f, -0.7071f},
                       { -0.7071f, 0.7071f, 0.0f }, };
```
Floats with lots of 9s should be checked if they round to a nicer number.